pkgs/servers/jicofo/default.nix

```diff
@@ -0,0 +1,47 @@
+{ pkgs, stdenv, fetchurl, dpkg, jre_headless, nixosTests }:
+
+let
+  pname = "jicofo";
+  version = "1.0-541";
+  src = fetchurl {
+    url = "https://download.jitsi.org/stable/${pname}_${version}-1_all.deb";
+    sha256 = "0s45bjsja2nkjhjcd2gx6hbby49v1d6r6553l6jfainycf6dh7xy";
+  };
+in
+stdenv.mkDerivation {
+  inherit pname version src;
+
+  phases = [ "unpackPhase" "installPhase" "fixupPhase" ];
+
+  unpackPhase = ''
```

Comment:
Please set `unpackCmd = "${dpkg}/bin/dpkg-deb -x $src .";` instead (and remove `dpkg` from `nativeBuildInputs)

Reply:
TIL. Changed it to `unpackCmd = "${dpkg}/bin/dpkg-deb -x $src debcontents";` to unpack into single directory.